dal/reddis.go

```diff
@@ -0,0 +1,55 @@
+package dal
+
+import (
+	"fmt"
+	"github.com/go-redis/redis"
+)
+
+func rClient() *redis.Client {
```

Comment:
why do you need the `r` prefix?

Reply:
redis client, in constrast to elasticsearch client for example